internal/common/mqtt/mqttconnection.go

```diff
@@ -67,14 +73,27 @@ func (client *Client) IsConnected() bool {
 
 // Disconnect disconnects the connection
 func (client *Client) Disconnect(quiesce uint) {
+	log.Info(logPrefix, "Disconnect requested from client")
 	if client.Client != nil {
 		client.Client.Disconnect(quiesce)
 	}
 }
 
-//StartMQTTClient is used to initiate the client and set the configuration
-func StartMQTTClient(brokerURL string) {
-
+// StartMQTTClient is used to initiate the client and set the configuration
+func StartMQTTClient(brokerURL string, clientID string) {
```

Comment:
I think the parameter for `brokerURL` is the IP or Domain name of host. 
It is confused with the result of [SetBrokerURL](https://github.com/lf-edge/edge-home-orchestration-go/blob/0589ede2610efbf9dd029d07a190ffe607d889d4/internal/common/mqtt/mqttconfig.go#L114). 
Please use other name instead of brokerURL to clarify the definition.